### DISCUSSION TITLE
After changing the region in the system Android, 1 appears next to the GitHub application

### DISCUSSION THREAD

**Discussion by olekstomek** [community/community#64508] [Mobile]
### Select Topic Area

Bug

### Body

In the GitHub mobile application for Android (currently I have version 1.127.0), after changing the region to any in Android system, the information "1" always appears next to the application icon, as if I had some activity to check. I have no notifications - the list is empty "All cauhgt up!".

Xiaomi Mi 9, Android 11 RKQ1.200826.002 (official)

**Comment by arafajs** [comment]
The issue is caused by a bug in the app that is counting the number of unread notifications in all of your repositories, even if they are in different regions. This is why the number "1" is always displayed, even if you have no notifications in the current region.

There is a workaround that you can try:

- Open the GitHub mobile app.
- Tap on the profile icon in the top left corner.
- Tap on Settings.
- Tap on Notifications.
- Toggle off the switch for "Show unread notifications in app icon."

This will disable the notification badge from being displayed. However, you will still receive notifications in the app.

**Reply by olekstomek** [reply]
I have a different view than you wrote. I have `Configure Notifications` and here I have options:
![1692631214106](https://github.com/community/community/assets/26818304/111d979a-932c-453d-b68c-be3be8a20261)
For me it is not a problem because the region does not change every day. I did this to check something else. Nevertheless, I decided that it's worth letting people know about this problem.

**Reply by arafajs** [reply]
I see. You are right, the options in your screenshot are different from the ones I mentioned. However, the issue is still the same. The app is counting the number of unread notifications in all of your repositories, even if they are in different regions. This is why the number "1" is always displayed, even if you have no notifications in the current region.
In the meantime, you can try the following to troubleshoot the issue:

- Clear the cache and data for the GitHub mobile app.
- Uninstall and reinstall the GitHub mobile app.
- Update the GitHub mobile app to the latest version.
- If you are still having trouble, then you can contact GitHub Support for help.

**Reply by olekstomek** [reply]
As I mentioned, the purpose of this submission is to add this issue to the backlog solution. This has a low impact on work difficulties. I also hope someone will take care of it. ;)
